apps/src/templates/CollapsibleSection.tsx

```diff
@@ -0,0 +1,84 @@
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import Typography from '@cdo/apps/componentLibrary/typography/Typography';
+import React, {useState} from 'react';
+import moduleStyles from './collapsible-section.module.scss';
+import {
+  SemanticTag as TypographyElementSemanticTag,
+  VisualAppearance as TypographyElementVisualAppearance,
+} from '@cdo/apps/componentLibrary/typography/types';
+
+interface CollapsibleSectionProps {
+  title: string;
+  titleSemanticTag?: TypographyElementSemanticTag;
+  titleVisualAppearance?: TypographyElementVisualAppearance;
+  titleStyle?: string;
+  titleIcon?: string;
+  titleIconStyle?: string;
+  children: React.ReactNode;
```

Comment:
minor readability nit: since there are so many optional props in this, could we move up `children` under `title` so the required props are at the top?